### PR TITLE
DOP-1128: adds path to rfc role to facilitate use

### PR DESCRIPTION
**Pull request by schmalliso**

### CODE REVIEW

File: snooty/rstspec.toml [hunk 1]
```diff
@@ -711,7 +711,7 @@ type = "text"
 
 [role.rfc]
 help = """Reference an IETF RFC."""
-type = "explicit_title"
+type = {link = "https://https://tools.ietf.org/html/%s"}
```
Comment (i80and) [inline snooty/rstspec.toml:714]: `https://https://`? :D

Comment (schmalliso) [reply snooty/rstspec.toml:714]: oh ffs.

Comment (schmalliso) [reply snooty/rstspec.toml]: Updated.

File: snooty/rstspec.toml [hunk 2]
```diff
@@ -711,7 +711,7 @@ type = "text"
 
 [role.rfc]
 help = """Reference an IETF RFC."""
-type = "explicit_title"
+type = {link = "https://tools.ietf.org/html/%s"}
```
Comment (i80and) [inline snooty/rstspec.toml:714]: Apparently IETF does have a redirect set up, but technically there should be an `rfc` prefix; i.e.
`/html/rfc%s` e.g. https://tools.ietf.org/html/rfc7919#appendix-A.2

**Pull request merged**